### PR TITLE
Task/moko 1438 moko bio

### PR DESCRIPTION
**Pull request by DenielGrin**
Add moko biometry

### CODE REVIEW

File: shared/build.gradle.kts [hunk 1]
```diff
@@ -37,27 +39,56 @@ kotlin {
                 implementation(compose.material)
                 @OptIn(org.jetbrains.compose.ExperimentalComposeLibrary::class)
                 implementation(compose.components.resources)
+                api("dev.icerock.moko:mvvm-core:0.15.0")
+                api("io.ktor:ktor-client-core:2.2.4")
+                api("dev.icerock.moko:network:0.20.1")
+                api("dev.icerock.moko:resources:0.20.1")
+//                api("dev.icerock.moko:resources-compose:0.20.1")
+
+                //Could not resolve all dependencies for configuration ':shared:iosSimulatorArm64CompileKlibraries'.
```
Comment (Alex009) [inline shared/build.gradle.kts:59]: library not support iosSimulator? can you check it in repo? and add issue link here

File: shared/build.gradle.kts [hunk 2]
```diff
@@ -37,27 +39,56 @@ kotlin {
...
             }
         }
+
         val androidMain by getting {
             dependencies {
                 api("androidx.activity:activity-compose:1.6.1")
                 api("androidx.appcompat:appcompat:1.6.1")
                 api("androidx.core:core-ktx:1.9.0")
+                api("io.ktor:ktor-client-okhttp:2.2.4")
+                api("dev.icerock.moko:biometry:0.2.0")
```
Comment (Alex009) [inline shared/build.gradle.kts]: it's not androidOnly lib

File: shared/src/androidMain/kotlin/main.android.kt [hunk 3]
```diff
@@ -1,5 +0,0 @@
-import androidx.compose.runtime.Composable
-
-actual fun getPlatformName(): String = "Android"
-
-@Composable fun MainView() = App()
```
Comment (Alex009) [inline shared/src/androidMain/kotlin/main.android.kt:5]: why it's deleted again?

**Review by Alex009 — CHANGES_REQUESTED** [review]

File: shared/src/androidMain/kotlin/main.android.kt (hunk 3 above)
Comment (DenielGrin) [reply shared/src/androidMain/kotlin/main.android.kt:5]: revert changes

File: shared/build.gradle.kts (hunk 2 above)
Comment (DenielGrin) [reply shared/build.gradle.kts]: cannot resolve these libraries in common code

**Comment by Alex009** [comment]
biometry is android & ios only. desktop not supported.

**Pull request closed without merging**